rclcpp/include/rclcpp/node.hpp

```diff
@@ -127,7 +128,9 @@ class Node
     size_t queue_size,
     std::function<void(const std::shared_ptr<MessageT> &)> callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr,
-    bool ignore_local_publications = false);
+    bool ignore_local_publications = false,
+    typename rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::SharedPtr
+    msg_mem_strat = rclcpp::msg_memory_strategy::MessageMemoryStrategy<MessageT>::create_default());
```

Comment:
You don't have to use indent-off for this. Sad that uncrustify is not handling it better. So be it.

Once the line gets shorted (when the ns goes away) it might be easier to read when wrapped after the equal sign.